network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html

```diff
@@ -33,8 +33,8 @@
       {% endfor %}
     </div>
     <div class="intro-button-wrapper">
-      <button class="btn btn-primary btn-newsletter d-lg-none mx-auto d-none">{% trans "Join Mozilla" %}</button>
-      <button class="btn btn-primary btn-newsletter d-none d-lg-block mx-auto d-none">{% trans "Join Mozilla" %}</button>
```

Comment:
We conditionally toggle `d-none` in the script depending on how far the page is scrolled. The new Bootstrap class combo `d-none d-lg-block` are creating issues with the script we set up. For simplicity, I'm replacing them with custom SCSS code.

Reply:
oh man well spotted